### ISSUE TITLE
matplotlib.pyplot.show() fails in pytorch container

### ISSUE THREAD

**Issue by xunleiwu** [pytorch/pytorch#37164]
## 🐛 Bug

matplotlib.pyplot.show() fails in pytorch container.

## To Reproduce

1. Launch pyTorch container
    docker run -e NVIDIA_VISIBLE_DEVICES=all -e DISPLAY=${DISPLAY} --ipc=host --rm -ti pytorch/pytorch:latest
GPU option does not really matter here.

2. Install matplotlib python library
pip install matplotlib

The container does not ship with matplotlib which is required in PyTorch 60 minute blitz tutorial.
https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html

3. Create a python script with below content
#!/opt/conda/bin/python

from __future__ import print_function
import torch as t
import torch.nn as nn
import torch.nn.functional as F
import torch.optim as optim
import torchvision as tv
import torchvision.transforms as transforms
import matplotlib.pyplot as plt
import numpy as np

# 1. Loading normalizing CIFAR10
batch_size = 32#4
num_workers = 32#2

transform = transforms.Compose(
    [transforms.ToTensor(),
     transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])

trainset = tv.datasets.CIFAR10(root='/tmp/data', train=True,
                                        download=True, transform=transform)
trainloader = t.utils.data.DataLoader(trainset, batch_size=batch_size,
                                          shuffle=True, num_workers=num_workers)

testset = tv.datasets.CIFAR10(root='/tmp/data', train=False,
                                       download=True, transform=transform)
testloader = t.utils.data.DataLoader(testset, batch_size=batch_size,
                                         shuffle=False, num_workers=num_workers)

classes = ('plane', 'car', 'bird', 'cat', 'deer', 'dog', 'frog', 'horse', 'ship', 'truck')

def imshow(img):
    img = img / 2 + 0.5     # unnormalize
    npimg = img.numpy()
    plt.imshow(np.transpose(npimg, (1, 2, 0)))
    plt.show()
    #plt.savefig('/tmp/image.png', bbox_inches='tight')

# get some random training images
dataiter = iter(trainloader)
images, labels = dataiter.next()

# show images
imshow(tv.utils.make_grid(images))
# print labels
print(' '.join('%5s' % classes[labels[j]] for j in range(4)))

4. By running the script inside the container, you shall see error:
UserWarning: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
  plt.show()

## Expected behavior
It is expected to see an image with 1x5 concatenated pictures from the dataset.
https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html

## Environment
$python collect_env.py
Collecting environment information...
PyTorch version: N/A
Is debug build: N/A
CUDA used to build PyTorch: N/A

OS: Red Hat Enterprise Linux Server release 7.7 (Maipo)
GCC version: (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)
CMake version: version 2.8.12.2

Python version: 3.4
Is CUDA available: N/A
CUDA runtime version: 10.2.89
GPU models and configuration:
GPU 0: Tesla V100-PCIE-16GB
GPU 1: Tesla V100-PCIE-16GB

Nvidia driver version: 440.33.01
cuDNN version: Probably one of the following:
/home/cas/daily/vb025/laxnd/191107/TKGrid/lib/libcudnn.so.7
/usr/local/cuda-10.2/targets/x86_64-linux/lib/libcudnn.so.7.6.5

Versions of relevant libraries:
[pip] Could not collect
[conda] Could not collect

## Additional context
Have tried to install tkagg inside the container but failed
# apt install python3-tk
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package python3-tk

**Comment by zhangguanheng66** [comment]
I think it's more relevant to our tutorial site and you may open an issue there.

https://github.com/pytorch/tutorials
